### ISSUE TITLE
Allow users to disable the "overwrite changes" button or another apparent way to get rid of it

### ISSUE THREAD

**Issue by tomtom** [syncthing/syncthing-android#1296]
For the last few versions of syncthing, syncthing for android sometimes asks "overwrite changes". This caused data loss once. It currently prevents syncthing from syncing a folder when it should actually copy the changes from the main computer. Also, there is now apparent way to get rid of this button. I changed the folder to read only on the main computer, I switched off "check for changes". Nothing helps.

Proposed change: In addition to "Overwrite changes" there should be a button for the other direction too "Locally overwrite changes".

**Comment by AudriusButkevicius** [comment]
Just don't press it.

**Comment by tomtom** [comment]
If I don't press the button, the folder doesn't get synced anymore.

**Comment by imsodin** [comment]
It does. Please use forum.syncthing.net for support.

**Comment by tomtom** [comment]
No it does not. 0kb are sent/received. Anyway, it's not my job to convince people who believe in their software that there is a problem.

**Comment by wweich** [comment]
The problem is that you misconfigured your folder. If the button is visible, it means you set the folder to send only, so no changes from other devices will be accepted.

**Comment by tomtom** [comment]
> If the button is visible, it means you set the folder to send only, so no changes from 
> other devices will be accepted.

I didn't give a full account. The folder worked some time ago. Then the 
folder was paused. I then reenabled the folder but the folder content 
was changed on the main computer. This is when I saw the button. I 
changed the folder on the main computer to send only. When editing the 
folder on android, I actually saw that the folder type was set to 
something unexpected - send only maybe. But it wasn't me who did this 
and the folder worked as expected before pausing it.

Anyway, you'll probably have to wait if anybody else reports a similar 
problem.
